core-build/gulp-core-build/src/index.ts

```diff
@@ -128,6 +133,8 @@ export function getConfig(): IBuildConfig {
  * @public
  */
 export function task(taskName: string, taskExecutable: IExecutable): IExecutable {
```

Comment:
Yes, but it will only be executed the first time it is called

---
In reply to: [131486331](https://github.com/Microsoft/web-build-tools/pull/294#discussion_r131486331) [](ancestors = 131486331)